content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -83,10 +83,19 @@ In the `.yaml` file for the Kubernetes object you want to create, you'll need to
 
 The precise format of the object `spec` is different for every Kubernetes object, and contains nested fields specific to that object. The [Kubernetes API Reference](https://kubernetes.io/docs/reference/kubernetes-api/) can help you find the spec format for all of the objects you can create using Kubernetes.
 
-For example, the reference for Pod details the [`spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
-for a Pod in the API, and the reference for Deployment details the [`spec` field](/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec) for Deployments.
-In those API reference pages you'll see mention of PodSpec and DeploymentSpec. These names are implementation details of the Golang code that Kubernetes uses to implement its API.
-
+For example, see the [`spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
+for the Pod API reference (implementation details of the Golang code that Kubernetes uses to implement its API).
```

Comment:
This is an improvement, but it's also slightly misleading. The Pod API reference is not “implementation details of the Golang code that Kubernetes uses to implement its API”. If someone wanted to write their own node implementation - some already exist - then those implementations are expected conform to the Pod API (and the Node API, etc) regardless of what programming language they use.

The _implementation details of the Golang code that Kubernetes uses to implement its API_ refers to the fragment identifier being named `PodSpec`, and that you see UpperCamelCase field names in that page - even though in the API it's not called a PodSpec; it's really a field named `spec` for resources of type `Pod`. In the Golang code for the kubelet, scheduler, etc, you do see `PodSpec` in upper camel case.

Reply:
I think the confusion comes from the subject being discussed (`spec` field) and the details of Pod API to which the parentheses refers to (unless I'm missing something). I'll remove the parentheses so there's no disambiguation.